webapp/pom.xml

```diff
@@ -41,6 +41,17 @@
     <dependency>
       <groupId>net.sourceforge.htmlcleaner</groupId>
       <artifactId>htmlcleaner</artifactId>
+      <scope>provided</scope>
+    </dependency>
+    <dependency>
+      <groupId>javax.portlet</groupId>
+      <artifactId>portlet-api</artifactId>
+      <scope>provided</scope>
+    </dependency>
+    <dependency>
+      <groupId>${project.groupId}</groupId>
+      <artifactId>news-services</artifactId>
+      <scope>provided</scope>
```

Comment:
you are in the same module news services, is this dependency really needed ?

Reply:
IMO I think it will be used by portlet